app-backup/borgbackup/borgbackup-9999.ebuild

```diff
@@ -44,4 +45,12 @@ DEPEND="
 src_install() {
 	distutils-r1_src_install
 	doman docs/man/*
+
+	dobashcomp scripts/shell_completions/bash/borg
```

Comment:
@mgorny Just in case, can you just try to emerge that ebuild that those files exist?